myapp/backend/backend.py

```diff
@@ -165,7 +165,7 @@ def get_team_roster():
         name = request.args.get('name')
         status = request.args.get('status')
         role = request.args.get('role')
-        position = request.args.get('position')
+        # position = request.args.get('position')
```

Comment:
Since 'position' isn't being used, would it be OK to remove the commented-out lines entirely?